### PR TITLE
Clean up Material UI components and relocate time ago icon

### PR DESCRIPTION
**Pull request by mirhamasala**
- https://howican.atlassian.net/browse/PER-67
- https://howican.atlassian.net/browse/PER-76

### CODE REVIEW

File: client/src/Components/NewsCard.js [hunk 1]
```diff
@@ -10,29 +10,41 @@ import CardContent from "@material-ui/core/CardContent";
...
+    paddingTop: "56.25%", // 16:9
   },
   title: {
-    fontSize: 13.5,
-    fontWeight: 500,
     color: "textPrimary",
-    textTransform: "uppercase",
-    paddingLeft: 0,
-    fontFamily: "Source Sans Pro",
+    fontFamily: "Avenir",
```
Comment (reneefromhold) [inline client/src/Components/NewsCard.js:31]: @mirhamasala Is the change in font related to the new design?

Comment (mirhamasala) [reply client/src/Components/NewsCard.js:31]: Yes.

I'd like to set this change site-wide. But I need to take the time to see how that ties in with Material UI.

**Comment by mirhamasala** [comment]
@reneefromhold, you can run the test manually.

```
cd client
npm test
```

**Comment by mirhamasala** [comment]
@reneefromhold As I understand it, `package.json` and `package-lock.json` should both be added to source control. Thus, **NOT** included in `gitignore`.

Source: [Should I include package-lock.json in source control?](https://www.ictshore.com/javascript/package-lock-json-source-control/)

**Pull request merged**